content/_index.md

```diff
@@ -3,34 +3,23 @@ title = "Stephen Augustus"
 
 He is the Head of [Open Source at Cisco][cisco-open-source], working within the
-Strategy, Incubation, & Applications (SIA) organization.
+Corporate Strategy, Development, and Incubation (CSDI) organization.
 
-For [Kubernetes][kubernetes], he has co-founded transformational elements of
-the project, including the [KEP][enhancements] (Kubernetes Enhancements
-Proposal) process, the [Release Engineering][releng] subproject, and Working
-Group Naming. Stephen has also previously served as a chair for both SIG PM and
-SIG Azure.
+Outside of managing Cisco's open source strategy and a team of brilliant architects in Cisco's Open Source Program Office (OSPO), Stephen participates in foundation-level governing boards (OpenSSF, CNCF, OpenAPI Initiative, project-level steering committees (Kubernetes, OpenSSF Scorecard, TODO Group), and helps maintain a few codebases around open source projects you may have heard of.
 
-He continues his work in Kubernetes as a [Steering Committee][k8s-steering]
-member and a Chair for [SIG Release][sig-release].
+Additionally, he serves as an advisor and invests in startups with open source in their DNA.
 
-Across the wider [LF][lf] (Linux Foundation) ecosystem, Stephen has the
-pleasure of serving as a member of the [OpenSSF][openssf]
-[Governing Board][openssf-gb] and the [OpenAPI Initiative][oapi-initiative]
-Business Governing Board.
+Within the Open Source Security Foundation, he maintains OpenSSF Scorecard, a project that can help you assess the security posture of open source projects.
 
-Previously, he was a [TODO Group][todo-group] [Steering Committee][todo-steering]
-member, a [CNCF][cncf] (Cloud Native Computing Foundation)
-[TAG Contributor Strategy][contrib-strat] Chair, and one of the Program Chairs
-for [KubeCon / CloudNativeCon][kubecon], the cloud native community's flagship
-conference.
+For [Kubernetes][kubernetes], Stephen serves as a [Steering Committee][k8s-steering] member and a Chair for [SIG Release][sig-release].
+He has co-founded transformational elements of the project, including the [KEP][enhancements] (Kubernetes Enhancements Proposal) process, the [Release Engineering][releng] subproject, and Working Group Naming. He has also previously served as a chair for both SIG PM and SIG Azure.
 
-He is a maintainer for the [Scorecard][scorecard] and [Dex][dex] projects, and
-a [prolific contributor][devstats] to CNCF projects, amongst the top 40
-(as of writing) code/content committers, all-time.
+Across the wider Cloud Native Computing Foundation ([CNCF][cncf]) ecosystem, he is a former [TAG Contributor Strategy][contrib-strat] Chair, [Dex][dex] maintainer, and Program Chair for [KubeCon / CloudNativeCon][kubecon], the cloud native community's flagship conference.
+
+Stephen is a [prolific contributor][devstats] to CNCF projects, amongst the top 100 (as of writing) code/content committers since inception of the foundation.
```

Comment:
```suggestion
Stephen is a [prolific contributor][devstats] to CNCF projects, and is amongst the top 100 (as of writing) code/content committers since the inception of the foundation.
```